### PR TITLE
Partial Filipino Translation - 1 112 translated words

### PR DESCRIPTION
**Pull request by mind-blast**
Partial Translation for advanced-merging.asc file - 1 112 translated words
Note : Code text and computer code text are not counted as translate words

### CODE REVIEW

File: book/07-git-tools/sections/advanced-merging.asc [hunk 1]
```diff
@@ -12,17 +12,17 @@ Samakatuwid, kung ikaw ay naghintay nang sobrang tagal upang i-merge ang dalawan
 Sa seksyon na ito, kami ay magpapatuloy kung ano ang mga maaaring isyu at anong mga kasangkapan na nagbibigay tulong upang panghawakan ang mga ito na mas madayang mga sitwasyon.
 At saka tatalakayin din natin ang iba't iba, na hindi pamantayan na mga uri ng mga merge na maaari mong gawin, pati na rin makita kung paano umatras sa mga merge na iyong nagawa.
 
-==== Merge Conflicts
+==== Pag-merge sa mga Kasalungat
 
-While we covered some basics on resolving merge conflicts in <<_git_branching#_basic_merge_conflicts>>, for more complex conflicts, Git provides a few tools to help you figure out what's going on and how to better deal with the conflict.
+Habang sakop namin ang maraming mga pangunahin na paglulutas ng merge na mga salungat sa <<_git_branching#_basic_merge_conflicts>>, para sa higit na kumplikado na mga salungat, Ang Git ay nagbibigay na ilang mga kasankapan para tulungan ka na tingnan kung ano ang nangyayari at kung paano mas mahusay na makitungo sa mga salungat.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/advanced-merging.asc]: - `Habang sakop namin ang maraming mga pangunahin na paglulutas` to `Habang nasakop natin ang maraming mga batayan sa paglulutas`
- `kasankapan` to `kasangkapan`

File: book/07-git-tools/sections/advanced-merging.asc [hunk 2]
```diff
@@ -12,17 +12,17 @@ Samakatuwid, kung ikaw ay naghintay nang sobrang tagal upang i-merge ang dalawan
...
 
-First of all, if at all possible, try to make sure your working directory is clean before doing a merge that may have conflicts.
-If you have work in progress, either commit it to a temporary branch or stash it.
-This makes it so that you can undo *anything* you try here.
-If you have unsaved changes in your working directory when you try a merge, some of these tips may help you lose that work.
+Una sa lahat, kung sa lahat ay posible, subukan na siguraduhin na ang iyong tinatrabahong direktoryo ay malinis bago magsagawa ng merge na maaaring magkakaroon ng mga salungat.
+Kung merong kang trabaho na nasa proseso, alinmang i-commit ito sa isamg pansamantalang branch o itago ito.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/advanced-merging.asc]: `isamg` to `isang`

File: book/07-git-tools/sections/advanced-merging.asc [hunk 3]
```diff
@@ -12,17 +12,17 @@ Samakatuwid, kung ikaw ay naghintay nang sobrang tagal upang i-merge ang dalawan
...
+Ginagawa ito nito para maka-undo ka ng *anumang bagay* na sinubukan mo dito.
+Kung meron kang hindi na-save na mga pagbabago sa iyong tintrabahong direktoryo kapag sinubukan mong i-merge, ilan sa mga tip na ito ay tumutulong sayo na mawala ang trabahong iyon.
 
-Let's walk through a very simple example.
-We have a super simple Ruby file that prints 'hello world'.
+Tingnan natin ng mabuti ang pinaka simpleng halimbawa.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/advanced-merging.asc]: change to `Tingnan nating mabuti ang pinakasimpleng halimbawa.`

File: book/07-git-tools/sections/advanced-merging.asc [hunk 4]
```diff
@@ -96,7 +96,7 @@ $ git commit -am 'document the function'
  1 file changed, 1 insertion(+)
 ----
 
-Now we try to merge in our `whitespace` branch and we'll get conflicts because of the whitespace changes.
+Ngayon ay sinusubukan namin na i-merge sa aming `whitespace` na branch at makakuha tayo ng mga salungat dahil sa whitespace na mga pagbabago.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/advanced-merging.asc]: Change `namin` and `aming` to `natin` and `ating`. This to ensure consistency when dealing when the pronouns such as 'we' in this context.

File: book/07-git-tools/sections/advanced-merging.asc [hunk 5]
```diff
@@ -107,11 +107,11 @@ Automatic merge failed; fix conflicts and then commit the result.
...
 [[_abort_merge]]
-===== Aborting a Merge
+===== Pag-abort ng Merge
 
-We now have a few options.
-First, let's cover how to get out of this situation.
-If you perhaps weren't expecting conflicts and don't want to quite deal with the situation yet, you can simply back out of the merge with `git merge --abort`.
+Tayo ngayon ay maroon nang ilang mga opsyon.
+Una, talakayin natin kung papaano makalabas sa sitwasyon na ito.
+King sasabihin mo na ikaw ay hindi umaasa ng mga salungat at hindi gusto ang medyong pakikitungo sa sitwasyon, maaari kang hindi magpatuloy sa pag-merge ng `git merge --abort`.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/advanced-merging.asc]: `Kung`

File: book/07-git-tools/sections/advanced-merging.asc [hunk 6]
```diff
@@ -125,21 +125,21 @@ $ git status -sb
 ## master
 ----
 
-The `git merge --abort` option tries to revert back to your state before you ran the merge.
-The only cases where it may not be able to do this perfectly would be if you had unstashed, uncommitted changes in your working directory when you ran it, otherwise it should work fine.
+Ang `git merge --abort` na opsyon na susubukang ibalik muli sa estado bago magpatakbo ng merge.
+Ang tanging mga kaso na kung saan ay hindi makagawa ng perpektong kung ito ay hindi naitago, ang mga hindi nai-commit na mga pagbabago sa iyong tinatrabahong direktoryo kapag pinatakbo mo ito, kung hindi man ito ay dapat gumagana.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/advanced-merging.asc]: - `hindi makagawa ng perpektong` to `hindi perpektong makagawa`
- `kung hindi man ito ay dapat gumagana` to `kung hindi man ay dapat itong gumana ng maayos`

File: book/07-git-tools/sections/advanced-merging.asc [hunk 7]
```diff
@@ -125,21 +125,21 @@ $ git status -sb
...
+Para sa kadahilanan na gusto mo lang magsimulang muli, maaari ka ding magpatakbo ng `git reset --hard HEAD`, at ang iyong repositoryo ay babalik sa huling na-commit na estado.
+Tandaan na anumang hindi nai-commit na trabaho ay mawawala, kaya siguraduhin mo ang anumang pagbabago.
 
-===== Ignoring Whitespace
+===== Hindi pinansin na Whitespace
 
-In this specific case, the conflicts are whitespace related.
-We know this because the case is simple, but it's also pretty easy to tell in real cases when looking at the conflict because every line is removed on one side and added again on the other.
-By default, Git sees all of these lines as being changed, so it can't merge the files.
+Sa ganitong partikular na kaso, ang mga salungat ay ang kaugnay sa whitespace.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/advanced-merging.asc]: may kaugnay sa whitespace

File: book/07-git-tools/sections/advanced-merging.asc [hunk 8]
```diff
@@ -150,28 +150,28 @@ Merge made by the 'recursive' strategy.
...
-What we really need to do is run the file we're trying to merge in through a `dos2unix` program before trying the actual file merge.
-So how would we do that?
+Ano talaga ang kailangan nating gawin ay ang patakbuhin ang file na sinusubukan nating i-merge sa pamamagitan ng isang `dos2unix` na programa bago sinusubukan ang aktwal file na nai-merge.
+Kaya paano natin ito gawin?
 
-First, we get into the merge conflict state.
-Then we want to get copies of my version of the file, their version (from the branch we're merging in) and the common version (from where both sides branched off).
-Then we want to fix up either their side or our side and re-try the merge again for just this single file.
+Una, kailangan natin pumunta sa merge na salungat na estado.
+Pagkatapos ay gusto naming kunin ang mga kopya sa aking bersyon sa file, ang kanilang bersyon(mula sa branch kami ay nagme-merge) at ang karaniwang bersyon (mula sa kung saang magkabilang panig na na-branch off).
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/advanced-merging.asc]: `naming` to `nating`

File: book/07-git-tools/sections/advanced-merging.asc [hunk 9]
```diff
@@ -150,28 +150,28 @@ Merge made by the 'recursive' strategy.
...
+Pagkatapos gusto naming ayusin ng maaga sa kanilang panig o sa aming panig at subukang muli ang pag-merge muli para sa solong file lamang.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/advanced-merging.asc]: `naming` to `nating`

File: book/07-git-tools/sections/advanced-merging.asc [hunk 10]
```diff
@@ -190,9 +190,9 @@ $ git ls-files -u
 100755 e85207e04dfdd5eb0a1e9febbc67fd837c44a1cd 3	hello.rb
 ----
 
-The `:1:hello.rb` is just a shorthand for looking up that blob SHA-1.
+Ang `:1:hello.rb` ay isa lamang shorthand para sa pagtingin na sa blob SHA-1.
 
-Now that we have the content of all three stages in our working directory, we can manually fix up theirs to fix the whitespace issue and re-merge the file with the little-known `git merge-file` command which does just that.
+Ngayon na mayroon kaming laman sa lahat ng tatlong mga stage sa aming tinatrabahong direktoryo, maaari naming mano-mano na ayusin ang kanilang aayusin na whitespace na isyu at i-merge uli ang file na may hindi gaaanong kilala `git merge-file` na utos na ginagawa lamang iyon.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/advanced-merging.asc]: `naming` to `nating`

File: book/07-git-tools/sections/advanced-merging.asc [hunk 11]
```diff
@@ -291,7 +291,7 @@ index ac51efd..44d0a25 100755
  hello()
 ----
 
-At this point we can use the `git clean` command to clear out the extra files we created to do the manual merge but no longer need.
+Mula rito pwede nating gamitin ang `git clean` na utos upang malinis ang dagdag na mga file na ginawa namin upang gawin ang manu-manong pag-merge ngunit hindi na kinakailangan.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/advanced-merging.asc]: `namin` to `natin`

**Review by josteinfrancois — COMMENTED** [review]
Hello @robin-ho . I have reviewed your work. Please see my comments. Thanks.

File: book/07-git-tools/sections/advanced-merging.asc (hunk 1 above)
Comment (mind-blast) [reply book/07-git-tools/sections/advanced-merging.asc]: done change

File: book/07-git-tools/sections/advanced-merging.asc (hunk 2 above)
Comment (mind-blast) [reply book/07-git-tools/sections/advanced-merging.asc]: done change

File: book/07-git-tools/sections/advanced-merging.asc (hunk 3 above)
Comment (mind-blast) [reply book/07-git-tools/sections/advanced-merging.asc]: done change

File: book/07-git-tools/sections/advanced-merging.asc (hunk 4 above)
Comment (mind-blast) [reply book/07-git-tools/sections/advanced-merging.asc]: done change

File: book/07-git-tools/sections/advanced-merging.asc (hunk 5 above)
Comment (mind-blast) [reply book/07-git-tools/sections/advanced-merging.asc]: done change

File: book/07-git-tools/sections/advanced-merging.asc (hunk 6 above)
Comment (mind-blast) [reply book/07-git-tools/sections/advanced-merging.asc]: done change

File: book/07-git-tools/sections/advanced-merging.asc (hunk 7 above)
Comment (mind-blast) [reply book/07-git-tools/sections/advanced-merging.asc]: done change

File: book/07-git-tools/sections/advanced-merging.asc (hunk 8 above)
Comment (mind-blast) [reply book/07-git-tools/sections/advanced-merging.asc]: done change

File: book/07-git-tools/sections/advanced-merging.asc (hunk 9 above)
Comment (mind-blast) [reply book/07-git-tools/sections/advanced-merging.asc]: done change

File: book/07-git-tools/sections/advanced-merging.asc (hunk 10 above)
Comment (mind-blast) [reply book/07-git-tools/sections/advanced-merging.asc]: done change

File: book/07-git-tools/sections/advanced-merging.asc (hunk 11 above)
Comment (mind-blast) [reply book/07-git-tools/sections/advanced-merging.asc]: done change

**Comment by mind-blast** [comment]
done change all suggested words.

**Pull request merged**